lib/acts_as_taggable_on/taggable/core.rb

```diff
@@ -65,6 +69,17 @@ def all_#{tags_type}_list
             def dirtify_tag_list(tagging)
               attribute_will_change! tagging.context.singularize+"_list"
             end
+              
+            def duplicated_mutations_from_database
+            unless defined?(@mutations_from_database)
+              @mutations_from_database = nil
+            end
+            @mutations_from_database ||= if defined?(@attributes)
+              ActiveModel::AttributeMutationTracker.new(@attributes)
+            else
+              NullMutationTracker.instance
+            end
+          end
```

Comment:
Indentation seems off for this method.

Reply:
Reverted because there was an extra commit.